src/App.tsx

```diff
@@ -63,6 +63,8 @@ const App = () => {
 
     const info = (
         <FlexBox $direction='column' $align='center'>
+            <Button size='small' label='uncheck all' onClick={reset} />
+            <Button size='small' label='check all' onClick={checkAll} />
```

Comment:
This won't work because this `info` is shared between `App` and `SideBar`:

![image](https://github.com/user-attachments/assets/45632051-06c5-462c-b0f3-4eb5a7bd0bf0)